src/changes-backwards-compatible.apib

```diff
@@ -6,23 +6,24 @@ We list all backwards-compatible additions here. These are currently available i
 
 #### Latest
 
+- In order to link milestones and deals to a task, we added `milestone_id` and `deal_id` as optional parameters to `tasks.create` and `tasks.update`.
 - We added `sent` as a boolean to `invoices.info` and `invoices.list`
 - We added an optional filter on `ids` to `invoices.list`.
 
 #### April 2019
 
 - In `users.info`, we added the `status` property to the response since it is now possible to retrieve deactivated users.
 
-- In `users.list`, you can now also request deactivated users, by using a new filter on `status`.  
```

Comment:
😞 your IDE is set to autoremove trailing spaces from code files, but in markdown that doesn't make sense, because that's a soft enter... could you revert this please?